### DISCUSSION TITLE
Align columns to tick marks

### DISCUSSION THREAD

**Discussion by Kaptensanders** [RomRider/apexcharts-card#616] [General]
Hi, in my graph displaying hourly energy prices from Nordpool, I get the columns centered over the x-axis labels. Making it look like the hourly price starts 30 min before it actually does. I want the columns left aligned with the tick marks.
Anyone have a suggestion?
 
![image](https://github.com/RomRider/apexcharts-card/assets/24979195/b600450a-6801-417f-9e4c-91502049347c)

```
      - type: custom:apexcharts-card
        graph_span: 24h
        header:
          show: true
          title: Energipris idag SEK/KWh
        span:
          start: day
        now:
          show: true
          label: Now
        yaxis:
          - decimals: 3
        hours_12: false

        series:
          - entity: sensor.nordpool
            type: column
            data_generator: |
              return entity.attributes.raw_today.map((start, index) => {
                return [new Date(start["start"]).getTime(), entity.attributes.raw_today[index]["value"]];
              });
            float_precision : 3

```

**Comment by Homeassister** [comment]
I was also looking for a solution to this problem. In the apexhcarts.js documentation there is an option for xaxis called "tickPlacement" for this, but it will not work in this card. I helped myself using the "time_delta" function.

.....
series:
   - entity: ......
     time_delta: +30min
     ....

The bars are then placed correctly. The only slight downside is that the displayed time is offset by 30 minutes when you click on a bar.

![Screenshot_20240314_102449](https://github.com/RomRider/apexcharts-card/assets/111651874/dae95eef-2de7-4dc9-b4f9-167cc05eee84)

![Screenshot_20240314_102557](https://github.com/RomRider/apexcharts-card/assets/111651874/2a332a29-35ac-4c2f-8594-146563732000)
